schemas/src/registry_schemas/schemas/name_request.json

```diff
@@ -0,0 +1,45 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "https://bcrs.gov.bc.ca/.well_known/schemas/name_request",
+    "type": "object",
+    "title": "Name Request Schema",
+    "properties": {
+        "nrNumber": {
+            "type": "string"
+        },
+        "nrName": {
+            "type": "string"
+        },
+        "legalType": {
+            "type": "string",
+            "title": "The Legal Type of the business.",
+            "default": "BC",
+            "enum": [
+                "BC",
+                "CP"
+            ]
+        },
+        "requestType": {
+            "type": "string"
+        },
+        "status": {
+            "type": "string"
+        },
+        "expDate": {
+            "type": "string",
+            "format": "date"
+        },
+        "condition": {
```

Comment:
This should be "consent" - the condition is wording in the NR decision text only (among other messaging in the text field) but this would be more around consent status. IF an entity requires consent from another organization to use a name, this is where it's tracked, and marked as received by staff when that happens. I'm not seeing where we're actually marking that in the new Names database (this team's last project) but it should come through as a single-character flag. You could leave it as unknown string for now but this might change.

Reply:
Ok sounds good, thanks for the heads up :) Like you said, the Schema is ever evolving!